src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -79,12 +80,17 @@ const ForgotPasswordPage = (props) => {
               </p>
               <LoginHelpLinks page="forgot-password" />
             </div>
-            <Button
+            <StatefulButton
+              type="submit"
               className="btn-primary submit"
+              state={status}
+              labels={{
+                null: intl.formatMessage(messages['logisration.forgot.password.page.submit.button']),
```

Comment:
If the label is the same, you don't need to specify it for each state. Specifying only `default` would work here.